packages/custom_lint/example/example_lint/bin/custom_lint_debug.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:io';
+import 'package:custom_lint/basic_runner.dart';
+
+Future<int> main() {
+  return runCustomLintOnDirectory(Directory.current.parent);
```

Comment:
Unfortunately you still have to have a separate debug main, in order to run / debug the custom linter, but the hot-reload logic will hot-reload any custom_lint plugins instead of just one. Additionally the logic is shared between the regular custom_lint runner and the debug. (This is the method that the custom_lint binary calls as well, but just with the current directory).

Reply:
The only way we could potentially work around this is if we figure out how to run custom_lint with the vm-service enabled. Currently doing a `pub global activate` doesn't seem to allow for custom arguments to the dart vm.